tardis/io/schemas/spectrum.yml

```diff
@@ -3,10 +3,13 @@ type: object
 properties:
   start:
     type: quantity
+    description: The beginning of the spectrum to be generated
   stop:
     type: quantity
+    description: The end of the spectrum to be generated
   num:
     type: number
+    description: Number of iterations used to build spectrum
```

Comment:
This is the number of bins that we use to generate the spectrum (however you want to phrase it)